packages/midi2-workbench/PKGBUILD

```diff
@@ -0,0 +1,66 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Florian Hülsmann <fh@cbix.de>
+
+_name=MIDI2.0Workbench
+pkgname=midi2-workbench
+pkgver=1.5.6
+_ver=${pkgver}p
+pkgrel=1
+pkgdesc='MIDI 2.0 debugging utility'
+arch=(x86_64)
```

Comment:
Arch Linux ARM doesn't have electron and I personally wouldn't bundle it only for that reason.